src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoData.cs

```diff
@@ -9,13 +9,12 @@ internal static class NumberFormatInfoData
         public static int[] UrINNumberGroupSizes()
         {
             if (
-                ((PlatformDetection.IsWindows && PlatformDetection.WindowsVersion >= 10)
+                (PlatformDetection.IsWindows && PlatformDetection.WindowsVersion >= 10)
 #if !uap
                 ||
                 (PlatformDetection.IsOSX && PlatformDetection.OSXKernelVersion >= new Version(15, 0))
 #endif
-                ) &&
-                (!PlatformDetection.IsUbuntu1510 && !PlatformDetection.IsUbuntu1604 && !PlatformDetection.IsUbuntu1610)
+                || PlatformDetection.IsUbuntu1510 || PlatformDetection.IsUbuntu1604 || PlatformDetection.IsUbuntu1610 || PlatformDetection.IsFedora
```

Comment:
@tarekgh I don't think Helix or Jenkins have runs for 1510 and i don't see it in our support matrix either. Just 1404, 1604, 1610. Perhaps we should remove IsUbuntu15010 everwhere.  @leecow ?